src/carplay.cs

```diff
@@ -573,11 +573,19 @@ interface CPListItem : CPSelectableListItem, NSSecureCoding {
 	interface CPListSection : NSSecureCoding {
 
 		[Export ("initWithItems:header:sectionIndexTitle:")]
+		[Obsolete ("Use 'CPListSection (ICPListTemplateItem [], string, string)' constructor instead.")]
 		NativeHandle Constructor (CPListItem [] items, [NullAllowed] string header, [NullAllowed] string sectionIndexTitle);
```

Comment:
Lets wrap the wrong API in !XAMCORE_5_0 so that we can remove it the next time we have breaking changes.
```suggestion
#if !XAMCORE_5_0
		[Export ("initWithItems:header:sectionIndexTitle:")]
		[Obsolete ("Use 'CPListSection (ICPListTemplateItem [], string, string)' constructor instead.")]
		NativeHandle Constructor (CPListItem [] items, [NullAllowed] string header, [NullAllowed] string sectionIndexTitle);
#endif
```